test/strategies/GQMF.DebugDeposit.Polygon.t.sol

```diff
@@ -6,12 +6,21 @@ import "../base/UniversalTest.sol";
 
 contract GammaQuickSwapMerklFarmStrategyTest is PolygonSetup, UniversalTest {
     constructor() {
-        vm.rollFork(53133804); // FAIL. Reason: revert: Improper ratio catched at this block with special amounts
+        // vm.rollFork(53_133_804); // FAIL. Reason: revert: Improper ratio catched at this block with special amounts
+        // assertEq(block.number, 53_133_804);
     }
 
-    function testGQMFDebugDeposit() public universalTest {
-        specialDepositAmounts.push(11838);
-        specialDepositAmounts.push(11971);
+    function testGQMFDebugDeposit(uint tokenA, uint tokenB) public universalTest {
+        // function testGQMFDebugDeposit() public universalTest {
+        vm.assume(tokenA < 20000);
+        vm.assume(tokenB < 20000);
+        vm.assume(tokenA > 10000);
```

Comment:
duplicate code?

Reply:
it's for sure the 20000> tokenA > 10000.